lib/Zonemaster/CLI.pm

```diff
@@ -192,7 +192,7 @@ has 'test' => (
     isa           => 'ArrayRef',
     required      => 0,
     documentation => __(
-'Specify test to run case-insensitively. Should be either the name of a module, or the name of a module and the name of a method in that module separated by a "/" character (Example: "Basic/basic01"). This switch can be repeated.'
+'Specify test to run case-insensitively. Should be either the name of a module, the name of a module and the name of a test case in that module separated by a "/" character (example: "Delegation/delegation01") or the name of a test case. This switch can be repeated.'
```

Comment:
I'm not particularly fond of this description so I tried to improve it. Here's my take:

```suggestion
'Specify test to run. Should be the case-insensitive name of a test module and/or a test case. (Examples: "Delegation", "delegation01", "Delegation/delegation01"). This switch can be repeated.'
```

Reply:
"Specify one or several test cases to be run."

Reply:
Updated with both suggestions.